<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Total</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
